lib/PhpBench.php

```diff
@@ -73,6 +73,10 @@ public static function normalizePath($path)
             return $path;
         }
 
+        if (PHP_OS === 'WINNT') {
+            return stream_resolve_include_path($path);
```

Comment:
I think instead of checking to see if the first char is `/` we should use the [webmozart/path-util](https://packagist.org/packages/webmozart/path-util) package:

```
if (Path::isAbsolute($path)) {
    return $path;
}
```

Reply:
I don't know the purpose of `normalizePath` well enough, but i guess it can be safely replaced by `stream_resolve_include_path`. At least, that's how phpunit resolves its test-direcotries.

Reply:
Well, it needs to make relative paths relative to where the command is executed. Without checking I am not sure to how many contexts this applies. Is there any use case for including things relative to all include paths?

If not I would prefer to just use the PathUtil fix above.

Reply:
I cannot think of any non-pathological use case. However, i'd say that `stream_resolve_include_path` produces the least surprising results because path-resolution then works in exactly the same way as for `require`, `fopen`, `file`, etc.

Reply:
The reason for using `getcwd` was to solve issues with resolving paths with the PHAR distribution. I will have to test the PHAR build with `stream_resolver_include_path`, and check exactly how PHPUnit does it.  However, I am still not sure if we want to be including things from all the include paths.

So, I am still leaning towards using the Path library as suggested above and keeping the existing `getcwd` logic.